metadata/email_addressesMetaData.php

```diff
@@ -80,13 +80,23 @@
             'default'        => 0,
             'vname'         => 'LBL_OPT_OUT',
         ),
+        'opt_in_email_created' => array(
+            'name' => 'opt_in_email_created',
+            'type' => 'datetime',
+            'vname' => 'LBL_OPT_IN_EMAIL_CREATED',
+        ),
```

Comment:
LBL_OPT_IN_EMAIL_CREATED seems a new language string (missing on language files)

Reply:
Oops. I have added in now. :)